django/db/models/query.py

```diff
@@ -607,22 +606,12 @@ def _extract_model_params(self, defaults, **kwargs):
                 ))
         return params
 
-    def _earliest_or_latest(self, *fields, field_name=None):
+    def _earliest_or_latest(self, *fields):
         """
         Return the latest object, according to the model's
         'get_latest_by' option or optional given field_name.
```

Comment:
This docstring needs updating:

`… field_name.` → `… fields.`

Reply:
Can you also correct it to be `… the earliest or latest object …`?

Reply:
I'm thinking that only one of "earlier" or "latest" is correct in the sense that this method itself doesn't handle the ordering. Maybe it should be "earliest" since `QuerySet.latest()` calls `reverse()` before chaining this method. Would you agree with renaming the method to `_earliest()` to make that more clear?

Reply:
Sounds reasonable to me. Thanks.

Reply:
Done in https://github.com/django/django/pull/10806.